docs/quickstarts/nextjs.mdx

```diff
@@ -21,6 +21,7 @@ description: Add authentication and user management to your Next.js app with Cle
 - Set up your environment keys to test your app locally
 - Protect your Next.js app's content with Clerk
 - Use Next.js middleware to implement route-specific authentication
+- Create a header with Clerk components for users to sign in and out

```

Comment:
The "you will learn" section should account for almost every heading/section of this tutorial.